.github/workflows/byob-gradle-trw.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 SLSA Authors
```

Comment:
let's rename this to `builder_gradle_slsa3.yml`, as it's the naming convention we follow for other builders.